src/MatchedLearnerApi.Types/PeriodDto.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace MatchedLearnerApi.Types
+{
+    public struct PeriodDto : IEquatable<PeriodDto>
+    {
+        public bool Equals(PeriodDto other)
+        {
+            return Period == other.Period && 
+                   IsPayable == other.IsPayable && 
+                   AccountId == other.AccountId && 
+                   ApprenticeshipId == other.ApprenticeshipId && 
+                   ApprenticeshipEmployerType == other.ApprenticeshipEmployerType && 
+                   TransferSenderAccountId == other.TransferSenderAccountId;
+        }
```

Comment:
From what I can tell the custom Equals behaviour here is solely for ignoring DataLockFailures in the unit tests, is this correct and if so why?